src/components/common/Map/Map.js

```diff
@@ -0,0 +1,51 @@
+import React, { useState, useEffect } from 'react';
+import { number, shape, bool } from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import GoogleMapReact from 'google-map-react';
+
+import { ReactComponent as LocationOval } from 'assets/oval_location.svg';
+import { ReactComponent as LocationIcon } from 'assets/icon_location.svg';
+import { DEFAULT_COORDINATES } from '../../../constants/constants';
```

Comment:
You should be able to use absolute import here

```suggestion
import { DEFAULT_COORDINATES } from 'constants/constants';
```